src/main/java/com/greenfoxacademy/kalendaryo/model/entity/GoogleCalendar.java

```diff
@@ -6,7 +6,9 @@
 public class GoogleCalendar {
```

Comment:
when we create the merged kalendar form already merged kalendars, we dont get the googleCalendarId, that we need